sopel/config/core_section.py

```diff
@@ -181,6 +185,15 @@ def homedir(self):
     capabilities.
     """
 
+    perform_commands = ListAttribute('perform_commands')
+    """A list of commands to perform upon successful connection to IRC server.
+
+    When entered using the config wizard, commas will be escaped automatically.
+    Otherwise, commas must be escaped, e.g.: `PRIVMSG Q@CServe.quakenet.org
+    :AUTH my_username MyPassword\,HasAComma@#$%!` Nothing else needs to be
+    escaped.
+    """  # noqa: W605; ignore escaped comma error
```

Comment:
This [isn't very future-proof](https://docs.python.org/3.6/reference/lexical_analysis.html#index-23).

> *Changed in version 3.6:* Unrecognized escape sequences produce a DeprecationWarning. In some future version of Python they will be a SyntaxError.

Instead of ignoring the warning, please fix it.

Reply:
Done.

Of the two options to resolve this:
1) convert to "raw" string literal
2) escape the `\`

I went with option (1), to avoid confusion. Assuming someone is reading the source to determine how to correctly escape commands, I imagine it would be easier to realize that a comma is escaped with a single `\`, instead of having to brain-parse the docstring to realize that `\\` actually refers to `\`, but is double-`\` only because the `\` needs to be escaped... 🤯